components/icu4x/Cargo.toml

```diff
@@ -5,7 +5,7 @@
 [package]
 name = "icu4x"
 description = "International Components for Unicode"
-version = "0.1.0"
+version = "1.0.0"
```

Comment:
this should be 1.0.1 right?

Reply:
(I want to make sure the published version matches stuff in tree)

in general in the future please don't publish without landing (or asking)